mobileconnect-example/src/main/AndroidManifest.xml

```diff
@@ -14,13 +14,13 @@
         android:name="com.telenor.connect.mobileconnectexample.ExampleApplication"
         android:supportsRtl="true"
         android:theme="@style/AppTheme" >
-        <activity android:name="com.telenor.connect.mobileconnectexample.SignedInActivity" />
-        <activity android:name="com.telenor.connect.mobileconnectexample.SignInActivity">
+        <activity android:name="com.telenor.connect.mobileconnectexample.SignedInActivity">
```

Comment:
It has changed (now the main activity is SignedInActivity (not SignInActivity as it was before)